card/src/main/java/com/adyen/checkout/card/util/AddressFormUtils.kt

```diff
@@ -57,7 +58,7 @@ internal object AddressFormUtils {
                     countryList
                 }
 
-                val defaultCountryCode = addressConfiguration.defaultCountryCode.orEmpty()
+                val defaultCountryCode = addressConfiguration.defaultCountryCode ?: shopperLocale.country.orEmpty()
```

Comment:
Maybe extract this `initialCountryCode` to a private function. In case we need to update the logic in the future it'll be easy to find.